pkgs.nix

```diff
@@ -55,17 +56,18 @@ let
 
   taxiZebra = pkgs.python3Packages.buildPythonPackage rec {
     pname = "taxi_zebra";
-    version = "2.3.1";
+    version = "3.0.1";
 
-    src = pkgs.python3.pkgs.fetchPypi {
-      inherit pname version;
-      sha256 = "177fzasgchgbixrr4xikfbis8i427qlyb8c93d404rjjny9g7nny";
+    src = pkgs.fetchFromGitHub {
```

Comment:
Is there a reason not to fetch the package from PyPI?

Reply:
This way we can run the tests when building the package.